<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieSearchWidget</class>
 <widget class="QWidget" name="MovieSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>661</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblScraper">
       <property name="text">
        <string>Scraper</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboScraper"/>
     </item>
     <item>
      <widget class="QLabel" name="lblLanguage">
       <property name="text">
        <string>Language</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LanguageCombo" name="comboLanguage">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::SizeAdjustPolicy::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="MyLineEdit" name="searchString">
     <property name="toolTip">
      <string>When using IMDb you can also use the IMDb id as search query.
If you want to search by an TMDb id please prefix it with &quot;id&quot;.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lblSuccessMessage">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblErrorMessage">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::PlainText</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
     <item>
      <layout class="QVBoxLayout" name="vboxResults">
       <item>
        <widget class="QLabel" name="lblCustomScraperInfo">
         <property name="text">
          <string notr="true">Custom Movie Scraper Infos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="opaqueResize">
          <bool>false</bool>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="QTableWidget" name="results">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Result</string>
           </property>
          </column>
         </widget>
         <widget class="ScrapePreview" name="preview" native="true"/>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vboxDetails">
       <item>
        <widget class="QGroupBox" name="detailsGroupBox">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>6</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="verticalLayout_41">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_42">
              <item>
               <widget class="MyCheckBox" name="chkTitle">
                <property name="text">
                 <string>Title</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyCheckBox" name="chkSet">
                <property name="text">
                 <string>Set</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyCheckBox" name="chkTagline">
                <property name="text">
                 <string>Tagline</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyCheckBox" name="chkRating">
                <property name="text">
                 <string>Rating</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyCheckBox" name="chkReleased">
                <property name="text">
                 <string>Released</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyCheckBox" name="chkRuntime">
                <property name="text">
                 <string>Runtime</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyCheckBox" name="chkDirector">
                <property name="text">
                 <string>Director</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyCheckBox" name="chkWriter">
                <property name="text">
                 <string>Writer</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyCheckBox" name="chkCertification">
                <property name="text">
                 <string>Certification</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyCheckBox" name="chkTrailer">
                <property name="text">
                 <string>Trailer</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyCheckBox" name="chkOverview">
                <property name="text">
                 <string>Overview</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyCheckBox" name="chkPoster">
                <property name="text">
                 <string>Poster</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_31">
                <property name="orientation">
                 <enum>Qt::Orientation::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_43">
              <item>
               <widget class="MyCheckBox" name="chkBackdrop">
                <property name="text">
                 <string>Backdrop</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyCheckBox" name="chkActors">
                <property name="text">
                 <string>Actors</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyCheckBox" name="chkGenres">
                <property name="text">
                 <string>Genres</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyCheckBox" name="chkStudios">
                <property name="text">
                 <string>Studios</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyCheckBox" name="chkCountries">
                <property name="text">
                 <string>Countries</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyCheckBox" name="chkTags">
                <property name="text">
                 <string>Tags</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyCheckBox" name="chkLogo">
                <property name="text">
                 <string>Logo</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyCheckBox" name="chkClearArt">
                <property name="text">
                 <string>Clear Art</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyCheckBox" name="chkCdArt">
                <property name="text">
                 <string>CD Art</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyCheckBox" name="chkBanner">
                <property name="text">
                 <string>Banner</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyCheckBox" name="chkThumb">
                <property name="text">
                 <string>Thumb</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_32">
                <property name="orientation">
                 <enum>Qt::Orientation::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkUnCheckAll">
            <property name="text">
             <string>(Un)Check all</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyCheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">ui/small_widgets/MyCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>ui/small_widgets/MyLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>LanguageCombo</class>
   <extends>QComboBox</extends>
   <header location="global">ui/small_widgets/LanguageCombo.h</header>
  </customwidget>
  <customwidget>
   <class>ScrapePreview</class>
   <extends>QWidget</extends>
   <header>ui/small_widgets/ScrapePreview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
